### DISCUSSION TITLE
3.0版LineChart，当数据变化，折线不断向左平移时，渐变不起作用

### DISCUSSION THREAD

**Discussion by XzavierDion** [XCharts-Team/XCharts#261] [Q&A]
3.0版LineChart，当数据变化，折线不断向左平移时，渐变失效，如何在动态下依然可以呈现折线渐变

**Comment by monitor1394** [comment]
目前是如何设置渐变的？是什么效果？

**Reply by XzavierDion** [reply]
chart.series[0].lineStyle.color = Color.red;
chart.series[0].lineStyle.toColor = Color.green;
我推测，由于我一直在不断的chart.AddData(1, data);添加数据。还限制了数据长度chart.SetMaxCache(500)。所以我的line chart颜色一直是red,没有渐变。

**Reply by XzavierDion** [reply]
我的需求就是一直添加数据

**Reply by monitor1394** [reply]
我测试了下是正常的，设置maxCache，平移后还是保持渐变的。

**Reply by XzavierDion** [reply]
还有一个问题，我这边一直添加数据，整个场景就会变得很卡，图表有哪里可以方便的定时清空数据吗？
